BAZProject/MVVM/Controllers/HomeController/HomeView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  HomeView.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 07/11/22.
+//
+
+import UIKit
+import Bond
+
+class HomeViewController: UIViewController {
+
+    @IBOutlet weak var movieTable: UITableView!
+    @IBOutlet weak var loadingView: UIActivityIndicatorView!
+    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet weak var myPicker: UIPickerView!
+    @IBOutlet weak var myToolbar: UIToolbar!
+    @IBOutlet weak var btnDone: UIBarButtonItem!
+    @IBOutlet weak var btnCancel: UIBarButtonItem!
+    
+    var viewModel: HomeViewModel?
+    lazy var alert: CustomAlertViewController = {
+        return CustomAlertViewController()
+    }()
+    
+    override func viewDidLoad() {
+        self.config()
+        self.setupObservables()
+        self.setupObservablesAction()
+    }
+    
+    func config() {
+        self.viewModel = HomeViewModel(usecase: ApiUseCase())
+        self.movieTable.register( UINib(nibName: MoviesCell.idReusable , bundle: nil) , forCellReuseIdentifier: MoviesCell.idReusable)
+        self.movieTable.delegate = self
+        self.movieTable.dataSource = self
+        self.searchBar.delegate = self
+        self.myPicker.delegate = self
+        self.myPicker.dataSource = self
+        self.viewModel?.getMovies()
```

Comment:
También aquí podríamos evitar el uso de `self` para que el código se vea más limpio
```suggestion
        movieTable.register( UINib(nibName: MoviesCell.idReusable , bundle: nil) , forCellReuseIdentifier: MoviesCell.idReusable)
        movieTable.delegate = self
        movieTable.dataSource = self
        searchBar.delegate = self
        myPicker.delegate = self
        myPicker.dataSource = self
        viewModel?.getMovies()
```